util/src/test/java/io/zeebe/util/SocketUtilTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.zeebe.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.net.Inet4Address;
+import java.net.Inet6Address;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.net.UnknownHostException;
+import org.junit.Test;
+
+public class SocketUtilTest {
+
+  @Test
+  public void shouldPrintDnsAndPort() {
+    final String host = "foo.barr";
+    final int port = 1234;
+    final String hostAndPortString =
+        SocketUtil.toHostAndPortString(InetSocketAddress.createUnresolved(host, port));
+    assertThat(hostAndPortString).isEqualTo(String.format("%s:%d", host, port));
```

Comment:
Let's just write "foo.barr:1234". Having hard coded here will help us better visualize what's the expected output. Plus the tests are simple/small enough that if we change the args we can easily change the values. Same for all the other tests.